src/components/Avatar/Avatar.component.tsx

```diff
@@ -199,44 +233,48 @@ export const Avatar: FC<AvatarProps> = ({
-          ambientLightIntensity={ambientLightIntensity}
-          dirLightPosition={dirLightPosition}
-          dirLightColor={dirLightColor}
-          dirLightIntensity={dirLightIntensity}
-          spotLightPosition={spotLightPosition}
-          spotLightColor={spotLightColor}
-          spotLightAngle={spotLightAngle}
-          spotLightIntensity={spotLightIntensity}
-          controlsMinDistance={
-            halfBody ? CAMERA.CONTROLS.HALF_BODY.MIN_DISTANCE : CAMERA.CONTROLS.FULL_BODY.MIN_DISTANCE
-          }
-          controlsMaxDistance={
-            halfBody ? CAMERA.CONTROLS.HALF_BODY.MAX_DISTANCE : CAMERA.CONTROLS.FULL_BODY.MAX_DISTANCE
-          }
-          updateCameraTargetOnZoom={!halfBody}
+    <BaseCanvas position={new Vector3(0, 0, 3)} fov={50} style={style} dpr={dpr} className={className}>
+      <Environment environment={environment} />
+      <CameraLighting
+        cameraTarget={cameraTarget}
+        cameraInitialDistance={cameraInitialDistance}
+        cameraZoomTarget={cameraZoomTarget}
+        ambientLightColor={ambientLightColor}
+        ambientLightIntensity={ambientLightIntensity}
+        dirLightPosition={dirLightPosition}
+        dirLightColor={dirLightColor}
+        dirLightIntensity={dirLightIntensity}
+        spotLightPosition={spotLightPosition}
+        spotLightColor={spotLightColor}
+        spotLightAngle={spotLightAngle}
+        spotLightIntensity={spotLightIntensity}
+        controlsMinDistance={halfBody ? CAMERA.CONTROLS.HALF_BODY.MIN_DISTANCE : CAMERA.CONTROLS.FULL_BODY.MIN_DISTANCE}
+        controlsMaxDistance={halfBody ? CAMERA.CONTROLS.HALF_BODY.MAX_DISTANCE : CAMERA.CONTROLS.FULL_BODY.MAX_DISTANCE}
+        updateCameraTargetOnZoom={!halfBody}
+      />
+      {AvatarModel}
+      {shadows && <Shadow />}
+      {background?.src && <Box {...background} />}
+      {capture && <Capture {...capture} />}
+      <EffectComposer disableNormalPass>
+        <Bloom
+          luminanceThreshold={bloom?.luminanceThreshold}
+          luminanceSmoothing={bloom?.luminanceSmoothing}
+          intensity={bloom?.intensity}
+          kernelSize={bloom?.kernelSize}
+          mipmapBlur={bloom?.mipmapBlur}
         />
-        {AvatarModel}
-        {shadows && <Shadow />}
-        {background?.src && <Box {...background} />}
-        {capture && <Capture {...capture} />}
-        <EffectComposer disableNormalPass>
-          <Bloom
-            luminanceThreshold={bloom?.luminanceThreshold}
-            luminanceSmoothing={bloom?.luminanceSmoothing}
-            intensity={bloom?.intensity}
-            kernelSize={bloom?.kernelSize}
-            mipmapBlur={bloom?.mipmapBlur}
-          />
-        </EffectComposer>
-      </BaseCanvas>
-    </Suspense>
+      </EffectComposer>
+    </BaseCanvas>
   );
 };
+
+const AvatarWrapper = (props: AvatarProps) => (
+  <Suspense fallback={props.loader ?? <Loader />}>
+    <Provider>
+      <Avatar {...props} />
+    </Provider>
+  </Suspense>
+);
```

Comment:
Why do we need this wrapper?

Reply:
To create a <Provider> to avoid the atom state to be shared between components, otherwise no matter how many visage you load all will do the same :)